OpenRA.Mods.Common/Traits/Harvester.cs

```diff
@@ -183,8 +183,8 @@ public Actor ClosestProc(Actor self, Actor ignore)
 
 					var occupancy = refs[loc].Occupancy;
 
-					// 4 harvesters clogs up the refinery's delivery location:
-					if (occupancy >= 3)
+					// 3 harvesters clogs up the refinery's delivery location:
+					if (occupancy >= 2)
```

Comment:
#3089 will disagree when this goes above than 2. I couldn't find the cause of that issue, but using xx== xxx= xx== foundations on the TS/RA2 refineries couldn't help. Although I guess it's related to the fact that sometimes when two harvesters arrive to the same refinery, one docks in to dump, then the other nudges it aside and the jamming maybe happens when a latecomer forces the already-at-docking position one to nudge away to a position which is already taken by the other latecomer.